### PR TITLE
Added binance vanilla options api paths

### PR DESCRIPTION
**Pull request by samgermain**
```
% binance vapiPublicGetTime                   
Debugger listening on ws://127.0.0.1:49681/c8100969-4af4-47bf-83de-61aa3c444ab9
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
2022-01-13T13:27:28.732Z
Node.js: v14.17.0
CCXT v1.67.31
binance.vapiPublicGetTime ()
1041 ms
{ code: '0', msg: 'success', data: '1642080450611' }
Waiting for the debugger to disconnect...
```

--------------------------

We'll have to figure how to record the [fee structure](https://www.binance.com/en/support/faq/374321c9317c473480243365298b8706)

> Fee Calculation
> Options fees have two parts: the transaction fee and the fee to exercise.
> Transaction fee will be charged after the transaction.
> Premiums are unidirectional, so if the buyer's profit when the Options is exercised is >0, the buyer will pay a fee.
> 
>  Transaction fee = Index price × Transaction fee rate.
>     Fee to exercise = Exercise price × Fee to exercise rate.
>     The transaction fee will not exceed 10% of the amount of the transaction.
>     The fee to exercise will not exceed 10% of the profit gained by exercising the Options.
> 
> Transaction fees for each transaction are calculated based on the index price at the time of order completion.
> Transaction fee rate：0.03% of the underlying value
> Exercise fee rate ： 0.1% of the underlying value

### CODE REVIEW

**Comment by kroitor** [comment]
Related: #11008

**Comment by samgermain** [comment]
@kroitor So the fees won't work correctly, seeing as the `option` bracket is empty

**Comment by kroitor** [comment]
> @kroitor So the fees won't work correctly, seeing as the `option` bracket is empty.

Can you please elaborate a bit? What exactly doesn't work?

**Comment by samgermain** [comment]
> > @kroitor So the fees won't work correctly, seeing as the `option` bracket is empty.
> 
> Can you please elaborate a bit? What exactly about `option` doesn't work?

The trading fees

File: js/binance.js [hunk 1]
```diff
@@ -710,6 +755,7 @@ module.exports = class binance extends Exchange {
                         },
                     },
                 },
+                'option': {},
```
Comment (samgermain) [inline js/binance.js:758]: This doesn't work

Comment (kroitor) [reply js/binance.js:758]: I don't really understand what you mean when you say _"doesn't work"_. What is expected, what do you get instead, can you fix/resolve it ?

Comment (kroitor) [reply js/binance.js:758]: I mean, see how it's done here for futures: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1546 < for we should copy the same approach for options, i think.

**Pull request merged**